inference-engine/samples/benchmark_app/main.cpp

```diff
@@ -528,7 +532,7 @@ int main(int argc, char* argv[]) {
 
         // Update number of streams
         for (auto&& ds : device_nstreams) {
-            const std::string key = ds.first + "_THROUGHPUT_STREAMS";
+            const std::string key = getDeviceTypeFromName(ds.first) + "_THROUGHPUT_STREAMS";
```

Comment:
Have you checked how it works when the common config is also loaded?

Benchmark_app has an opportunity to load JSON config with settings. But there is a rule, that command-line setting has the first priority. What we will have if the user loads the config with a number of streams N for GPU in config (no device id), but specifies M and K streams for GPU.0 and GPU.1 in the command line? I don't see the code changes which can handle this situation. Could you please check this scenario?